### PR TITLE
[i18n-fr] Translate installation to French

### PR DESCRIPTION
**Pull request by NoB0**
# What does this PR do?

Translated the quicktour.mdx file of the documentation to French.

Part of #21456

Thank you in advance for your review.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

French speaking contributors. 
Documentation: @stevhliu and @MKhalusova

### CODE REVIEW

**Comment by CharbelAD** [comment]
Heads up, a couple of typos I was able to spot:

- line 31: missing 's' ->  environnements
- line 57: en seule ligne -> en une seule ligne
- line 74 : suivant -> suivants
- line 110: utilie -> utile

**Comment by NoB0** [comment]
> Heads up, a few typos I was able to spot:
> 
> * line 31: missing 's' ->  environnements
> * line 57: en seule ligne -> en une seule ligne
> * line 74 : suivant -> suivants
> * line 110: utilie -> utile

Thanks for pointing this out! I corrected them and check for others, I believe it is now ok.

**Comment by MKhalusova** [comment]
Thank you for working on this! Please make sure the CI checks are green. You can find more information here https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md

**Review by MKhalusova — APPROVED** [review]
Thank you for the translation! Looks good to me as far as structure goes, and to my limited understanding of French.

File: docs/source/fr/installation.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Installation
+
+Installez 🤗 Transformers pour la librairie d'apprentissage profond avec laquelle vous travaillez, configurer votre cache et configurer 🤗 Transformers pour un usage hors ligne (facultatif).
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:19]:
```suggestion
Installez 🤗 Transformers pour n'importe quelle librairie d'apprentissage profond avec laquelle vous avez l'habitude de travaillez, configurez votre cache et configurez 🤗 Transformers pour un usage hors ligne (facultatif).
```

File: docs/source/fr/installation.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+  * Instructions d'installation pour [PyTorch](https://pytorch.org/get-started/locally/).
+  * Instructions d'installation pour [TensorFlow 2.0](https://www.tensorflow.org/install/pip).
+  * Instructions d'installation pour [Flax](https://flax.readthedocs.io/en/latest/).
+
+## Installation avec pip
+
+Vous devriez installer 🤗 Transformers dans un [environnement virtuel](https://docs.python.org/3/library/venv.html).
+Si vous n'êtes pas à l'aise avec les environnements virtuels, consultez ce [guide](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/).
+Utiliser un environnement virtuel permet de facilement gérer différent projets et d'éviter des erreurs de compatibilité entre les différentes dépendances.
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:32]:
```suggestion
Utiliser un environnement virtuel permet de facilement gérer différents projets et d'éviter des erreurs de compatibilité entre les différentes dépendances.
```

File: docs/source/fr/installation.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+🤗 Transformers et TensorFlow 2.0 :
+
+```bash
+pip install 'transformers[tf-cpu]'
+```
+
+<Tip warning={true}>
+
+Utilisateurs avec M1 / ARM
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:73]:
```suggestion
Pour le architectures mac M1 / ARM
```

File: docs/source/fr/installation.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+
+</Tip>
+
+🤗 Transformers et Flax :
+
+```bash
+pip install 'transformers[flax]'
+```
+
+Vérifiez si 🤗 Transformers a bien été installé avec la commande suivante. La commande va télécharger un modèle pré-entraîné :
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:90]:
```suggestion
Vérifiez que 🤗 Transformers a bien été installé avec la commande suivante. La commande va télécharger un modèle pré-entraîné :
```

File: docs/source/fr/installation.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+
+```bash
+python -c "from transformers import pipeline; print(pipeline('sentiment-analysis')('we love you'))"
+```
+
+Le label et score est ensuite affiché :
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:96]:
```suggestion
Le label et score sont ensuite affichés :
```

File: docs/source/fr/installation.md [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+```bash
+pip install git+https://github.com/huggingface/transformers
+```
+
+Cette commande installe la version depuis la branche `main` au lieu de la dernière version stable.
+La version de la branche `main` est utile pour avoir les derniers développements.
+Par exemple, si un bug a été résolu depuis la dernière version stable mais n'a pas encore été publié officiellement.
+Cependant, cela veut aussi dire que la version de la branche `main` n'est pas toujours stable.
+Nous nous efforçons de maintenir la version de la branche `main` opérationnelle, et la plupart des problèmes sont généralement résolus en l'espace de quelques heures ou d'un jour.
+Si vous recontrez un problème, n'hésitez pas à créer une [Issue](https://github.com/huggingface/transformers/issues) pour que l'on puisse trouver une solution au plus vite !
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:115]: does not need to be split into lines

File: docs/source/fr/installation.md [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+
+Vérifiez si 🤗 Transformers a bien été installé avec la commande suivante :
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:117]:
```suggestion
Vérifiez que 🤗 Transformers a bien été installé avec la commande suivante :
```

File: docs/source/fr/installation.md [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+Clonez le projet et installez 🤗 Transformers avec les commandes suivantes :
+
+```bash
+git clone https://github.com/huggingface/transformers.git
+cd transformers
+pip install -e .
+```
+
+Ces commandes créent des liens entre le dossier où le projet a été cloné et les chemins de vos librairies Python.
+Python regardera maintenant dans le dossier que vous avez cloné en plus des chemins de vos librairies normaux.
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:139]: to rephrase, les chemins des librairies normaux doesn't have much sense

File: docs/source/fr/installation.md [hunk 9]
```diff
@@ -0,0 +1,265 @@
...
+</Tip>
+
+Maintenant, vous pouvez facilement mettre à jour votre clone avec la dernière version de 🤗 Transformers en utilisant la commande suivante :
+
+```bash
+cd ~/transformers/
+git pull
+```
+
+Votre environnement Python trouvera la version de la branche `main` lors de la prochaine exécution.
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:155]:
```suggestion
Votre environnement Python utilisera la version de la branche `main` lors de la prochaine exécution.
```

File: docs/source/fr/installation.md [hunk 10]
```diff
@@ -0,0 +1,265 @@
...
+
+## Installation avec conda
+
+Installation avec la chaîne `huggingface` de conda :
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:159]: Mmm the channel does not translate to `chaine` in this case but rather `canal`

File: docs/source/fr/installation.md [hunk 11]
```diff
@@ -0,0 +1,265 @@
...
+
+```bash
+conda install -c huggingface transformers
+```
+
+## Configuration du cache
+
+Les modèles pré-entraînés sont téléchargés et cachés localement dans le dossier suivant : `~/.cache/huggingface/hub`. C'est le dossier par défaut donné par la variable d'environnement `TRANSFORMERS_CACHE`. Sur Windows, le dossier par défaut est `C:\Users\nom_utilisateur\.cache\huggingface\hub`. Vous pouvez modifier les variables d'environnement indiquées ci-dessous - par ordre de priorité - pour spécifier un dossier de cache différent :
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:167]:
```suggestion
Les modèles pré-entraînés sont téléchargés et mis en cache localement dans le dossier suivant : `~/.cache/huggingface/hub`. C'est le dossier par défaut donné par la variable d'environnement `TRANSFORMERS_CACHE`. Sur Windows, le dossier par défaut est `C:\Users\nom_utilisateur\.cache\huggingface\hub`. Vous pouvez modifier les variables d'environnement indiquées ci-dessous - par ordre de priorité - pour spécifier un dossier de cache différent :
```

File: docs/source/fr/installation.md [hunk 12]
```diff
@@ -0,0 +1,265 @@
...
+
+🤗 Transformers peut fonctionner dans un environnement cloisonné ou hors ligne en n'utilisant que des fichiers locaux. Définissez la variable d'environnement `TRANSFORMERS_OFFLINE=1` pour activer ce mode.
+
+<Tip>
+
+Ajoutez [🤗 Datasets](https://huggingface.co/docs/datasets/) à votre processus d'entraînement hors ligne en définissant la variable d'environnement `HF_DATASETS_OFFLINE=1`.
+
+</Tip>
+
+Par example, vous exécuteriez généralement un programme sur un réseau normal cloisonné des instances externes avec la commande suivante :
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:189]: complicated way of saying that you are running something under proxy

File: docs/source/fr/installation.md [hunk 13]
```diff
@@ -0,0 +1,265 @@
...
+```bash
+HF_DATASETS_OFFLINE=1 TRANSFORMERS_OFFLINE=1 \
+python examples/pytorch/translation/run_translation.py --model_name_or_path t5-small --dataset_name wmt16 --dataset_config ro-en ...
+```
+
+Le script devrait maintenant s'exécuter sans rester en attente ou attendre une expiration, car il sait qu'il ne doit rechercher que des fichiers locaux.
+
+### Récupérer des modèles et des tokenizers pour une utilisation hors ligne
+
+Une autre option pour utiliser 🤗 Transformers hors ligne est de télécharger les fichiers à l'avance, puis d'utiliser vers leur chemin local lorsque vous en avez besoin en mode hors ligne. Il existe trois façons de faire cela :
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:206]:
```suggestion
Une autre option pour utiliser 🤗 Transformers hors ligne est de télécharger les fichiers à l'avance, puis d'utiliser les chemins locaux lorsque vous en avez besoin en mode hors ligne. Il existe trois façons de faire cela :
```

File: docs/source/fr/installation.md [hunk 14]
```diff
@@ -0,0 +1,265 @@
...
+
+```py
+>>> from transformers import AutoConfig
+
+>>> config = AutoConfig.from_pretrained("./your/path/bigscience_t0/config.json")
+```
+
+<Tip>
+
+Consultez la section [How to download files from the Hub](https://huggingface.co/docs/hub/how-to-downstream) pour plus de détails sur le téléchargement de fichiers stockés sur le Hub.
```
Comment (ArthurZucker) [inline docs/source/fr/installation.md:263]:
```suggestion
Consultez la section [How to download files from the Hub (Comment télécharger des fichiers depuis le Hub)](https://huggingface.co/docs/hub/how-to-downstream) pour plus de détails sur le téléchargement de fichiers stockés sur le Hub.
```

**Review by ArthurZucker — COMMENTED** [review]
Thanks for translating

File: docs/source/fr/installation.md (hunk 6 above)
Comment (NoB0) [reply docs/source/fr/installation.md:115]: Removed new lines.

File: docs/source/fr/installation.md (hunk 8 above)
Comment (NoB0) [reply docs/source/fr/installation.md:139]: I was not sure how to translate that accurately, I made a new suggestion.

File: docs/source/fr/installation.md (hunk 12 above)
Comment (NoB0) [reply docs/source/fr/installation.md:189]: It appears that is particular sentence was removed from the English documentation. I updated the content to match the English documentation. I am a bit unsure about the translation of the last sentence (i.e., `Seuls les fichiers ...`), please check.

**Review by ArthurZucker — APPROVED** [review]
Thanks for the translation!

**Pull request merged**